docs/tech/logging.md

```diff
@@ -0,0 +1,15 @@
+# Logging
+
+All logging must happen to a standardized path, under `/var/praekelt/logs`. If your app
+has a large amount of files (celery, worker, and app, for instance), write each of
+these to a named path under `/var/praekelt/logs/appname/foo.log`, otherwise just
+`/var/praekelt/logs/foo.log` is sufficient.
+
+Logs always end with the extension `.log`. `.err` is not valid. If you need to write out
+error logs, either use the name `foo-error.log`, or write your supervisord configuration
+with the `redirect_stderr` option.
```

Comment:
Yeah, this is from https://ways-of-working.readthedocs.io/en/latest/platforms.html#logging - I removed a lot of the stuff about how we deploy Django (because I assume that's also containers) but I didn't know if the logging was still relevant. Will remove!